cognite_toolkit/_cdf_tk/data_classes/_manifest.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+import sys
+from dataclasses import dataclass, field
+from pathlib import Path
+
+if sys.version_info >= (3, 11):
+    import toml
+else:
+    import tomli as toml
+
+
+@dataclass(frozen=True)
+class Manifest:
+    version: str
+    description: str | None
+    tags: list[str] | None = field(default_factory=list)
```

Comment:
No need to have `None` as it will now default to an empty list.
```suggestion
    tags: list[str] = field(default_factory=list)
```